cern-notebook/tests/test_notebook.py

```diff
@@ -0,0 +1,52 @@
+import os
+import subprocess
+import tempfile
+import nbformat
+
+cur_path = os.path.abspath(".")
+notebooks_path = os.path.join(cur_path, "notebooks")
+kernels = ["ir"]
```

Comment:
This properly needs to be updated to `kernels = ["python3"]`

Reply:
Fixed